ipa-multipoint/ipa_multipoint_jni/Cargo.toml

```
@@ -0,0 +1,24 @@
+# SPDX-License-Identifier: Apache-2.0
+
+[package]
+name = "ipa_multipoint_jni"
+version = "0.1.0"
+description = "IPA Multipoint Crypto Library"
+license = "Apache-2.0"
+authors = ["Antoine Toulme <antoine@lunar-ocean.com>"]
+repository = "https://github.com/hyperledger/besu-native"
+edition = "2018"
+
+[dependencies]
+ipa-multipoint = { path = "../ipa_multipoint" }
```

I recommend using the `git =` key so that it will go get the latest version. This repo is mostly stable, but some tweaks are still in the pipeline.

```suggestion
ipa-multipoint = { git = "https://github.com/crate-crypto/ipa_multipoint", branch = "develop" }
```

---

fixed